File: 박유정/9월/4주차/0919/12982.py
def solution(d, budget):
    count = 0
    for i in range(0,len(d)):
        budget = budget - min(d)
        count += 1
        d.remove(min(d))
        
        if budget<0:
            count -= 1
            break
    
    answer = count
    return answer

File: 구은성/11월/1104/12982.java
import java.util.*;
import java.util.stream.Collectors;

class Solution {
    public int solution(int[] d, int budget) {
        int answer = 0;
        List<Integer> list = Arrays.stream(d)
            .boxed()
            .collect(Collectors.toList());
        
        list.sort(Comparator.naturalOrder());
        
        int sum = 0;
        for(int num : list) {
            sum += num;
            if (sum > budget) {
                break;
            }
            else {
                answer++;
            }
        }
        
        return answer;
    }
}